packages/@css-blocks/ember-utils/src/options.ts

```diff
@@ -5,12 +5,63 @@ import type { OptiCSSOptions } from "opticss";
 
 type Writeable<T> = { -readonly [P in keyof T]: T[P] };
 
+/**
+ * Sourcemap options provided by fast-sourcemap-concat, which is used
+ * by broccoli-concat to build sourcemaps. This is *not* an exhaustive
+ * list, so a catch-all is provided for undocumented props if needed.
+ */
+export interface BroccoliConcatSourcemapOptions {
+  enabled?: boolean;
+  extensions?: string[];
+  inline?: boolean;
+  mapCommentType?: "block" | "line";
+  [propName: string]: unknown;
+}
+
+/**
+ * Options provided by broccoli-concat, which the ember-app plugin uses
+ * to combine CSS files together during the post-process step. You can
+ * use these options to control the order in which files are concatenated,
+ * or other options related to concatenation.
+ */
+export interface BroccoliConcatOptions {
+  outputFile?: string;
+  header?: string;
+  headerFiles?: string[];
+  /**
+   * The files to contatenate together during the postprocess step. Do keep
+   * in mind this you'll have access only to the files in the CSS postprocess
+   * tree for ember-app. You can use globs if preferred.
+   *
+   * Any top-level files that are processed by Ember CLI will likely be stored
+   * at "styles/<name-of-file>.css". You can use a plugin such as broccoli-debug
+   * to determine the structure of your styles after processing.
+   *
+   * Your CSS Blocks file will be available by default
+   * at "styles/css-blocks.css", but will vary if the "output" property is set
```

Comment:
I thought these files were in the `assets` directory instead of the `styles` directory... that's what we're using in the config defaults at least.

```suggestion
   * at "assets/<name-of-file>.css". You can use a plugin such as broccoli-debug
   * to determine the structure of your styles after processing.
   *
   * Your CSS Blocks file will be available by default
   * at "assets/css-blocks.css", but will vary if the "output" property is set
```